doc/build/faq/performance.rst

```diff
@@ -271,7 +271,7 @@ Below is a simple recipe which works profiling into a context manager::
 To profile a section of code::
 
     with profiled():
-        Session.query(FooClass).filter(FooClass.somevalue==8).all()
+        session.execute(select(FooClass).where(FooClass.somevalue==8)).all()
```

Comment:
any time there is `session.query(OneClass).all()`, that is, query against a single full class object, these should all use `session.scalars(select(OneClass)).all()`, so people are encouraged to get lists of FooClass objects as is what they usually want

it looks like that's how you have it for most of them so maybe this one is an oversight